en/docs/administer/http-access-logging.md

```diff
@@ -12,29 +12,23 @@ details.
 
 ### Configuring access logs for the HTTP servlet transport
 
-As the runtime of WSO2 Identity Server is based on Apache Tomcat, you can use
-the `         Access_Log_Valve        ` variable in Tomcat as explained
-below to configure access logs to the HTTP servlet transport:
-
-1.  Open the `<IS_HOME>/repository/conf/tomcat/catalina-server.xml          `
-    file (which is the server descriptor file for the embedded Tomcat
-    integration)
-
-2.  Customize the attributes for the
-    `           Access_Log_Valve          ` variable shown below.
-
-    ``` java
-    <Valve className="org.apache.catalina.valves.AccessLogValve"
-    directory="${carbon.home}/repository/logs"
-    prefix="localhost_access_log_sample."
-    suffix=".log"
-    pattern="combined"
+As the runtime of WSO2 Identity Server is based on Apache Tomcat,`Access_Log_Valve` variable 
+configured in `<IS_HOME>/repository/conf/tomcat/catalina-server.xml` handles the access logs to the HTTP servlet 
```

Comment:
Replace with deployment.toml.

Reply:
I left that on purpose. This just explains where this thing are handled but when configuring they are guided towards toml file

Reply:
removed